design/process/review-guidelines.md

```diff
@@ -8,40 +8,55 @@ editor: markdown
   - an expected timeline for the review, if any.
 - Be sure to include the specific feedback you are seeking.
-- Tag the required reviewers by their GitHub username.
-  - Always tag the Mathesar members with Product or Design roles (see [Team](/team) for handles). If the issue is related to implementation, tag the members with Engineering roles as well.
-- Please post the GitHub discussion link to `#design-review:matrix.mathesar.org`. 
-- If the reviewers request changes or have questions, please make the changes and re-request the review by posting on the GitHub Discussion and tagging reviewers.
-  - Make sure to summarize your changes in your comment.
-- Once everyone has had a chance to review (or the timeline expires) and you're satisfied with the feedback, update the document and ensure that it all fits together well.
-
-[Here's an example review request for reference](https://github.com/centerofci/mathesar/discussions/305).
-
-## Reviewers
-- Check for outstanding design review requests at least once a day.
-- Comment on the GitHub Discussion with feedback. Please follow the general advice above.
-- Once you're done reviewing the document, if you don't have any feedback, post on the discussion and say you're done reviewing.
-- Specify explicitly if you want another look at the design spec before it is finalized.
+- Assign the required reviewers by their GitHub username.
+  - Always request a review from the Mathesar members with Product or Design roles (see [Team](/team) for handles.
+  - If the issue contains technical implementation details, you can request a review from a member with Engineering role as well. Keep in mind that a review from a single Engineering role member should be enough for most design issues. If you are tagging more than one Engineering member, specify the type of feedback that is required from them.
+- Please post the GitHub PR link to `#design-review:matrix.mathesar.org`.
+- If the reviewers request changes or have questions, please make the changes and re-request the review.
+- Once everyone has had a chance to review (or the timeline expires) and you're satisfied with the feedback, update the spec and ensure that it all fits together nicely.
+
+### Reviewers
+
+- Check for outstanding [design spec review requests](https://github.com/centerofci/mathesar-wiki/pulls?q=is%3Aopen) at least once a day.
+- Comment on the GitHub PR with feedback. Please follow the general advice above.
+Once you're done reviewing the document, approve the changes or unassign yourself from the PR if you don't have any feedback.
+- Request changes explicitly if you want them included in the spec before it is finalized.
 - By sharing feedback, you actively participate in the UX process; make sure you read the context documents and are clear on the user's needs. If needed, ask questions, don't make assumptions.
 
-# Advice for Reviewers
+#### Advice for Reviewers
```

Comment:
Please have this be a top level heading so that it's easier to find from the table of contents on the wiki.